ui/src/Explore/__snapshots__/CommitLog.test.tsx.snap

```diff
@@ -107,3 +107,17 @@ exports[`Commit log renders 2`] = `
   </div>
 </DocumentFragment>
 `;
+
+exports[`Commit redirects on an invalid ref 1`] = `
+<DocumentFragment>
+  <div
+    class="EmptyView card-header"
+  >
+    <span
+      class="EmptyView__message"
+    >
+      Nothing to show
+    </span>
+  </div>
+</DocumentFragment>
```

Comment:
Why this change? Is it related to the "content" renaming?

Reply:
it was auto-generated after running `./mvnw clean install`

Reply:
could you move it to a separate PR (before this one)? This PR is already huge and these UI changes are not related...

Reply:
yet, it's not a big deal :)